modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -271,9 +271,9 @@ protected String commandKraft() {
     }
 
     protected String commandZookeeper() {
-        String command = "echo 'clientPort=" + ZOOKEEPER_PORT + "' > zookeeper.properties\n";
-        command += "echo 'dataDir=/var/lib/zookeeper/data' >> zookeeper.properties\n";
-        command += "echo 'dataLogDir=/var/lib/zookeeper/log' >> zookeeper.properties\n";
+        String command = "echo 'clientPort=" + ZOOKEEPER_PORT + "' > /tmp/zookeeper.properties\n";
+        command += "echo 'dataDir=/var/lib/zookeeper/data' >> /tmp/zookeeper.properties\n";
+        command += "echo 'dataLogDir=/var/lib/zookeeper/log' >> /tmp/zookeeper.properties\n";
         command += "zookeeper-server-start zookeeper.properties &\n";
```

Comment:
I had to change this to ` command += "zookeeper-server-start /tmp/zookeeper.properties &\n";`